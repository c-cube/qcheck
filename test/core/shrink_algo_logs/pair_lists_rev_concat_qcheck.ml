open QCheck


let pair_lists_rev_concat =
  Test.make ~name:"pairs lists rev concat"
    (pair (list pos_int) (list pos_int))
    (Log.shrinks Print.(pair (list int) (list int)) @@
     fun (xs,ys) -> List.rev (xs@ys) = (List.rev xs)@(List.rev ys))
;;
try Test.check_exn pair_lists_rev_concat with Test.Test_fail _ -> ()
