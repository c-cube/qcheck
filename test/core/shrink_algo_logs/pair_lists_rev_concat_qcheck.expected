fails ([746128036134350390; 3320138915538235609; 762885526058710783; 2576722397014108197; 2212906866981369341; 1640905365439660237; 1332813644634888458; 4200732135473036391; 4019205362601190422; 3604960216201531334; 1030920758746055813; 42698023673707078; 461254803691789422; 3907752289102763354; 2376625294767365140; 233912705210533500; 2885984270693699778; 206688355204064373; 3125861600804315502; 995060844299483603; 3304051089873692429; 2844386052677950810; 783948934796125836; 2811664358749514936; 577396296345547726; 4034593187793338681; 2876338321387969175; 87742541397720461; 3913144465737681491; 2157897674888391252; 1900045916102699699; 470374935996346701; 1058614341825424502; 1559319129724948356; 3213658892819147890; 3499789205307451184; 2949893178354239875; 4081767524822084600; 3498215066845906710; 3888114807866831417; 1617765899064123599; 222148284913888236; 1727736514015983487; 3627298274602852316; 3898699378202599689; 2325974463411213714; 736613994040121347; 3414272520188964400; 2385629250013173880; 1422734660007259092; 4444687725003531440; 457048644972362124; 4167435680361369008; 3782659039753844321; 3535545447366295718; 1427021198443879304; 1780554213351577732; 2339368131399213148; 2751299455197144887; 2600977953485488800; 2162723098335521495; 1983575711044503816; 2874018304296889121; 1748110412865819786; 3323702044602342403; 1051679191852119048; 2091273115503484635; 1494934802956191728; 2924085192404355751; 3504533652220676065; 1021194546620295917; 1475531882113471101; 2107955955816427122; 4212121560373082225; 469399435806458576; 2088479218423533002; 1863269600650047234; 2442470379891932470; 2578459975428991434; 3314506194157773646; 2099966072041048397; 4345233542768063678; 2336561938922647935; 3244060950469953135; 4079854335523721792; 2828045190569874047; 1785372679127697414; 4540055182899246774; 1752752047792425988; 701266008688472284; 3366925537143040935; 1598238962204476610; 2927976139793784142; 323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2385629250013173880; 1422734660007259092; 4444687725003531440; 457048644972362124; 4167435680361369008; 3782659039753844321; 3535545447366295718; 1427021198443879304; 1780554213351577732; 2339368131399213148; 2751299455197144887; 2600977953485488800; 2162723098335521495; 1983575711044503816; 2874018304296889121; 1748110412865819786; 3323702044602342403; 1051679191852119048; 2091273115503484635; 1494934802956191728; 2924085192404355751; 3504533652220676065; 1021194546620295917; 1475531882113471101; 2107955955816427122; 4212121560373082225; 469399435806458576; 2088479218423533002; 1863269600650047234; 2442470379891932470; 2578459975428991434; 3314506194157773646; 2099966072041048397; 4345233542768063678; 2336561938922647935; 3244060950469953135; 4079854335523721792; 2828045190569874047; 1785372679127697414; 4540055182899246774; 1752752047792425988; 701266008688472284; 3366925537143040935; 1598238962204476610; 2927976139793784142; 323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2107955955816427122; 4212121560373082225; 469399435806458576; 2088479218423533002; 1863269600650047234; 2442470379891932470; 2578459975428991434; 3314506194157773646; 2099966072041048397; 4345233542768063678; 2336561938922647935; 3244060950469953135; 4079854335523721792; 2828045190569874047; 1785372679127697414; 4540055182899246774; 1752752047792425988; 701266008688472284; 3366925537143040935; 1598238962204476610; 2927976139793784142; 323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([4079854335523721792; 2828045190569874047; 1785372679127697414; 4540055182899246774; 1752752047792425988; 701266008688472284; 3366925537143040935; 1598238962204476610; 2927976139793784142; 323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([3366925537143040935; 1598238962204476610; 2927976139793784142; 323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([75857926498367735], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([37928963249183868], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([18964481624591934], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([9482240812295967], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([4741120406147984], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2370560203073992], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([1185280101536996], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([592640050768498], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([296320025384249], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([148160012692125], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([74080006346063], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([37040003173032], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([18520001586516], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([9260000793258], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([4630000396629], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2315000198315], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([1157500099158], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([578750049579], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([289375024790], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([144687512395], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([72343756198], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([36171878099], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([18085939050], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([9042969525], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([4521484763], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2260742382], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([1130371191], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([565185596], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([282592798], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([141296399], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([70648200], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([35324100], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([17662050], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([8831025], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([4415513], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2207757], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([1103879], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([551940], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([275970], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([137985], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([68993], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([34497], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([17249], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([8625], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([4313], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2157], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([1079], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([540], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([270], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([135], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([68], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([34], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([17], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([9], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([5], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([3], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([1], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([0], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([0], [1035416029544138122; 1118378519987614091])
holds ([], [1035416029544138122; 1118378519987614091])
fails ([0], [1118378519987614091])
holds ([], [1118378519987614091])
holds ([0], [])
fails ([0], [559189259993807046])
holds ([], [559189259993807046])
holds ([0], [])
fails ([0], [279594629996903523])
holds ([], [279594629996903523])
holds ([0], [])
fails ([0], [139797314998451762])
holds ([], [139797314998451762])
holds ([0], [])
fails ([0], [69898657499225881])
holds ([], [69898657499225881])
holds ([0], [])
fails ([0], [34949328749612941])
holds ([], [34949328749612941])
holds ([0], [])
fails ([0], [17474664374806471])
holds ([], [17474664374806471])
holds ([0], [])
fails ([0], [8737332187403236])
holds ([], [8737332187403236])
holds ([0], [])
fails ([0], [4368666093701618])
holds ([], [4368666093701618])
holds ([0], [])
fails ([0], [2184333046850809])
holds ([], [2184333046850809])
holds ([0], [])
fails ([0], [1092166523425405])
holds ([], [1092166523425405])
holds ([0], [])
fails ([0], [546083261712703])
holds ([], [546083261712703])
holds ([0], [])
fails ([0], [273041630856352])
holds ([], [273041630856352])
holds ([0], [])
fails ([0], [136520815428176])
holds ([], [136520815428176])
holds ([0], [])
fails ([0], [68260407714088])
holds ([], [68260407714088])
holds ([0], [])
fails ([0], [34130203857044])
holds ([], [34130203857044])
holds ([0], [])
fails ([0], [17065101928522])
holds ([], [17065101928522])
holds ([0], [])
fails ([0], [8532550964261])
holds ([], [8532550964261])
holds ([0], [])
fails ([0], [4266275482131])
holds ([], [4266275482131])
holds ([0], [])
fails ([0], [2133137741066])
holds ([], [2133137741066])
holds ([0], [])
fails ([0], [1066568870533])
holds ([], [1066568870533])
holds ([0], [])
fails ([0], [533284435267])
holds ([], [533284435267])
holds ([0], [])
fails ([0], [266642217634])
holds ([], [266642217634])
holds ([0], [])
fails ([0], [133321108817])
holds ([], [133321108817])
holds ([0], [])
fails ([0], [66660554409])
holds ([], [66660554409])
holds ([0], [])
fails ([0], [33330277205])
holds ([], [33330277205])
holds ([0], [])
fails ([0], [16665138603])
holds ([], [16665138603])
holds ([0], [])
fails ([0], [8332569302])
holds ([], [8332569302])
holds ([0], [])
fails ([0], [4166284651])
holds ([], [4166284651])
holds ([0], [])
fails ([0], [2083142326])
holds ([], [2083142326])
holds ([0], [])
fails ([0], [1041571163])
holds ([], [1041571163])
holds ([0], [])
fails ([0], [520785582])
holds ([], [520785582])
holds ([0], [])
fails ([0], [260392791])
holds ([], [260392791])
holds ([0], [])
fails ([0], [130196396])
holds ([], [130196396])
holds ([0], [])
fails ([0], [65098198])
holds ([], [65098198])
holds ([0], [])
fails ([0], [32549099])
holds ([], [32549099])
holds ([0], [])
fails ([0], [16274550])
holds ([], [16274550])
holds ([0], [])
fails ([0], [8137275])
holds ([], [8137275])
holds ([0], [])
fails ([0], [4068638])
holds ([], [4068638])
holds ([0], [])
fails ([0], [2034319])
holds ([], [2034319])
holds ([0], [])
fails ([0], [1017160])
holds ([], [1017160])
holds ([0], [])
fails ([0], [508580])
holds ([], [508580])
holds ([0], [])
fails ([0], [254290])
holds ([], [254290])
holds ([0], [])
fails ([0], [127145])
holds ([], [127145])
holds ([0], [])
fails ([0], [63573])
holds ([], [63573])
holds ([0], [])
fails ([0], [31787])
holds ([], [31787])
holds ([0], [])
fails ([0], [15894])
holds ([], [15894])
holds ([0], [])
fails ([0], [7947])
holds ([], [7947])
holds ([0], [])
fails ([0], [3974])
holds ([], [3974])
holds ([0], [])
fails ([0], [1987])
holds ([], [1987])
holds ([0], [])
fails ([0], [994])
holds ([], [994])
holds ([0], [])
fails ([0], [497])
holds ([], [497])
holds ([0], [])
fails ([0], [249])
holds ([], [249])
holds ([0], [])
fails ([0], [125])
holds ([], [125])
holds ([0], [])
fails ([0], [63])
holds ([], [63])
holds ([0], [])
fails ([0], [32])
holds ([], [32])
holds ([0], [])
fails ([0], [16])
holds ([], [16])
holds ([0], [])
fails ([0], [8])
holds ([], [8])
holds ([0], [])
fails ([0], [4])
holds ([], [4])
holds ([0], [])
fails ([0], [2])
holds ([], [2])
holds ([0], [])
fails ([0], [1])
holds ([], [1])
holds ([0], [])
holds ([0], [0])
